python/llm/example/GPU/HF-Transformers-AutoModels/Model/redpajama/README.md

```diff
@@ -1,4 +1,4 @@
-# redpajama
+# Redpajama
```

Comment:
RedPajama, P also needs to be capitalized

Reply:
OK.